pkg/apis/kops/cluster.go

```diff
@@ -67,6 +67,9 @@ type ClusterSpec struct {
 	// MasterInternalName is the internal DNS name for the master nodes
 	MasterInternalName string `json:"masterInternalName,omitempty"`
 
+	// AdditionalSANs adds additional Subject Alternate Names to apiserver cert that kops generates
+	AdditionalSANs []string `json:"additionalSans,omitempty"`
```

Comment:
So this adds the API value.  Where is the biz logic to add the SAN to the cert?  I am probably missing it.